web/src/app/users/UserContactMethodCreateDialog.tsx

```diff
@@ -35,7 +35,7 @@ const userConflictQuery = gql`
 `
 
 export default function UserContactMethodCreateDialog(props: {
-  userID: string
+  userID: string | null
```

Comment:
I feel like a `userID` is always required to create a contact method -- is there type information elsewhere that is missing that should be fixed instead?